dashi/rnaseqc/over_time.py

```diff
@@ -132,44 +315,102 @@ def create_subplot(rna_df, graph_list):
+                    value=ALL_KITS
                 ),
                 html.Br(),
                 html.Label('Dates: '),
                 dcc.DatePickerRange(
                     id='date_picker',
-                    min_date_allowed=min(rna_df['Run Date']),
-                    max_date_allowed=max(rna_df['Run Date']),
-                    start_date=min(rna_df['Run Date']),
-                    end_date=max(rna_df['Run Date']),
+                    display_format='YYYY-MM-DD',
+                    min_date_allowed=min(RNA_DF['Run Date']),
+                    max_date_allowed=max(RNA_DF['Run Date']),
+                    start_date=min(RNA_DF['Run Date']),
+                    end_date=max(RNA_DF['Run Date']),
                 ),
                 html.Br(),
+                html.Br(),
                 html.Label('Show Graphs:'),
                 dcc.Dropdown(
                     id='graphs_to_plot',
                     multi=True,
-                    options=[{'label': x, 'value': x} for x in graphs_to_plot],
-                    value=graphs_to_plot[:4]
+                    options=[
+                        {'label': x, 'value': x} for x in METRICS_TO_GRAPH
+                    ],
+                    value=METRICS_TO_GRAPH[:4]
                 ),
+                html.Br(),
+                html.Div([
+                    html.Div([
+                        html.Label('Colour by:'),
+                        dcc.Dropdown(
+                            id='colour_by',
+                            options=[
+                                {'label': x['name'], 'value': x['id']} for x
+                                in SHAPE_COLOUR_COLUMN
+                            ],
+                            value='Study Title'
+                        ),
+                    ], style={'width': '45%', 'display': 'inline-block', }),
+                    html.Div([
+                        html.Label('Shape by:'),
+                        dcc.Dropdown(
+                            id='shape_by',
+                            options=[
+                                {'label': x['name'], 'value': x['id']} for x
+                                in SHAPE_COLOUR_COLUMN
+                            ],
+                            value='geo_prep_kit'
+                        ),
+                    ], style={
+                        'width': '45%', 'display': 'inline-block',
+                        'float': 'right'
+                    }),
+                ])
             ], style={'margin': '23px'})]
         ),
-        sd_material_ui.RaisedButton(id='filter_button', label='Filters'),
+        html.Div([
+            html.Div(
+                sd_material_ui.RaisedButton(
+                    id='filter_button', label='Options'
+                ), style={'display': 'inline-block', }
+            ),
+            html.Div(
+                sd_material_ui.RaisedButton(
+                    id='update_from_table_button', label='Update Order',
```

Comment:
`Update Order` is vague. Does this mean `re-graph`?

Reply:
Changed to `Update Graphs` bce269f. Note this button is used to update the graphs when the sorting of the DataTable is changed. You can change the sorting by clicking the small arrows in the header. Clicking the button makes the graphs re-plot given that order.